src/components/FeatureViews/Accounts/UserProfile.js

```diff
@@ -229,8 +230,17 @@ class UserProfile extends Component {
     renderNoteIcon() {
         if (this.getUser() && this.getUser().role != "receptionist") {
             if (this.hasImportantNotes()) {
+                const numImportantNotes = Object.values(this.getUser().notes)
+                    .filter(({important}) => important).length;
```

Comment:
Can be replaced with a `reduce`: `.reduce((total, {important}) => important ? total + 1 : total, 0)`